pixiedust/services/stashCloudant.py

```diff
@@ -19,11 +19,20 @@
 import time
 import requests
 import json
+import pixiedust
+from pyspark import SparkContext
 
 CLOUDANT_CONN_TYPE = "cloudant"
-
+myLogger = pixiedust.getLogger(__name__)
 class StashCloudantHandler(Display):
     def doRender(self, handlerId):
+        sc = SparkContext.getOrCreate()
+        config = sc._conf.getAll()
+        
+        sc.stop()
```

Comment:
Stopping the SparkContext is bad, there is only one for the whole session. Can you please remove this call.